doc/source/tune/api/syncing.rst

```diff
@@ -12,6 +12,6 @@ Tune Syncing Configuration
 --------------------------
 
 .. autosummary::
-    :toctree: doc/
 
     ray.train.SyncConfig
+        :noindex:
```

Comment:
what's the reason why we are no indexing here?

Reply:
It's indexed in the Train API now